rpcs3/rpcs3qt/kernel_explorer.cpp

```diff
@@ -167,17 +168,20 @@ kernel_explorer::kernel_explorer(QWidget* parent) : QDialog(parent)
 	m_tree->header()->close();
 
 	// Merge and display everything
-	vbox_panel->addSpacing(10);
+	vbox_panel->addSpacing(8);
 	vbox_panel->addLayout(hbox_buttons);
-	vbox_panel->addSpacing(10);
+	vbox_panel->addSpacing(8);
 	vbox_panel->addWidget(m_tree);
-	vbox_panel->addSpacing(10);
 	setLayout(vbox_panel);
 
 	// Events
 	connect(button_refresh, &QAbstractButton::clicked, this, &kernel_explorer::Update);
+}
 
-	Update();
+void kernel_explorer::closeEvent(QCloseEvent *event)
```

Comment:
I still don't understand why you don't just use the done signal or whatever fits best in main_window